README.md

```diff
@@ -149,13 +135,13 @@ Se asume que la persona tiene conocimientos previos en:
 
 1. Añadir como colaboradores (ir a settings del repositorio y en Collaborators) a:
    * [dianakrog](https://github.com/dianakrog)
-   * [Scot3004](https://github.com/Scot3004)
    * [kliver98](https://github.com/kliver98)
+   * [diegomtylop](https://github.com/diegomtylop)
```

Comment:
This is in another PR https://github.com/testing-community/rest-assured-workshop/pull/7

Reply:
update with main